lib/_http_server.js

```diff
@@ -223,7 +225,8 @@ function writeHead(statusCode, reason, obj) {
   }
 
   if (common._checkInvalidHeaderChar(this.statusMessage))
-    throw new Error('Invalid character in statusMessage.');
+    throw new errors.Error('ERR_HTTP_INVALID_CHAR', 'Invalid character in ' +
```

Comment:
I'm guessing its broken into 2 strings to avoid the long line.  I'm not sure I've seen us use the + for this, instead could you just put the 'Invalid ...' on a second line